src/main/nsis/bin/process.bat

```diff
@@ -24,6 +24,25 @@ goto end
 set PROCESS_OPTION=
 if not [%2]==[] set PROCESS_OPTION=process.name=%2
 
-..\Java\bin\java.exe -cp ..\${pom.build.finalName}-uber.jar -Dsalesforce.config.dir=%1 com.salesforce.dataloader.process.ProcessRunner %PROCESS_OPTION%
+
+IF "%JAVA_HOME%" == "" (
+    for /f "tokens=*" %%i in ('${pom.build.finalName}-java-home.exe') do (
+        IF EXIST "%%i" (
+            set JAVA_HOME=%%i
+        ) ELSE (
+            echo %%i
+        )
+    )
+)
+
+IF "%JAVA_HOME%" == "" (
+    echo Please set JAVA_HOME before running the encrypt.bat
```

Comment:
No.. line 33 will say something like "This application requires JAVA 1.8 32bit" but it won't say anything about JAVA_HOME. 

We can omit it since if the user doesn't set the system will try to figure it out. I'll ask Tess if it's noise for users.